mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -612,39 +619,34 @@ def main(args=None):
 
     paths = create_paths(parsed_args.mlir_build_dir, parsed_args.miopen_build_dir)
     archNames = getArch()
-    xdlops = False
-    for x in archNames:
-        if "gfx908" in x or "gfx90a" in x:
-            xdlops = True
-
     arch = ','.join(archNames)
     configs = getConfigurations(paths.configuration_file_path)
 
-
     #If no arguments are passed, then benchmark with MLIR and MIOpen
     if parsed_args.batch_both:
         # batch benchmark with MLIR and MIOpen.
-        generatePerformanceResults(configs, xdlops, paths, arch)
+        generatePerformanceResults(configs, paths, arch)
     elif parsed_args.miopen_use_tuned_mlir:
-        benchmarkMIOpenWithMLIRKernels(configs, xdlops, reportUtils.MIOPEN_TUNED_REPORT_FILE, paths)
+        benchmarkMIOpenWithMLIRKernels(configs, arch, reportUtils.MIOPEN_TUNED_REPORT_FILE, paths)
     elif parsed_args.miopen_use_untuned_mlir:
-        benchmarkMIOpenWithMLIRKernels(configs, xdlops, reportUtils.MIOPEN_UNTUNED_REPORT_FILE, paths)
+        benchmarkMIOpenWithMLIRKernels(configs, arch, reportUtils.MIOPEN_UNTUNED_REPORT_FILE, paths)
     elif parsed_args.tuning:
-        tuneMLIRKernels(configs, xdlops, paths)
+        tuneMLIRKernels(configs, paths, arch)
     else:
         if parsed_args.batch_mlir:
-            df = pd.DataFrame(benchmarkMLIR(testVector.split(sep=' '), xdlops, paths, arch) for testVector in configs)
+            df = pd.DataFrame(benchmarkMLIR(testVector.split(sep=' '), paths, arch) for testVector in configs)
         elif parsed_args.batch_miopen:
-            df = pd.DataFrame(benchmarkMIOpen(testVector.split(sep=' '), xdlops, paths) for testVector in configs)
+            df = pd.DataFrame(benchmarkMIOpen(testVector.split(sep=' '), paths, arch) for testVector in configs)
         elif parsed_args.miopen:
-            df = pd.DataFrame([benchmarkMIOpen(parsed_args.config, xdlops, paths)])
+            df = pd.DataFrame([benchmarkMIOpen(parsed_args.config, paths, arch)])
         else:
             # Will only reach here with more than 1 unspecified arguments
             # These are arguments are directly passed through to benchmarkMLIR
             if not parsed_args.mlir_build_dir:
                 raise RuntimeError("MLIR build dir was not provided/found")
-            df = pd.DataFrame([benchmarkMLIR(parsed_args.config, xdlops, paths, arch)])
-        df.to_csv(parsed_args.fileName)
+            df = pd.DataFrame([benchmarkMLIR(parsed_args.config, paths, arch)])
+        chip = GFX_CHIP_RE.search(arch).group(0)
+        df.to_csv(chip + '_' + parsed_args.fileName)
```

Comment:
Also, instead of doing this, please adjust the defaults for `filename` to include `chip`. I want `-o` to mean `-o`, not some weird hybrid of `-o` plus this behavior